downloader/src/main/kotlin/Downloader.kt

```diff
@@ -333,7 +334,9 @@ object Downloader {
                 .build()
 
             try {
-                OkHttpClientHelper.execute(request).use { response ->
+                OkHttpClientHelper.execute(request) {
+                    authenticator(Authenticator.JAVA_NET_AUTHENTICATOR)
```

Comment:
Is there a reason not to do this in the `OkHttpClientHelper` itself (and adjust the comment over there in that regard)?

Reply:
I considered it and the comment made me extra careful.
If you think it has its place globally in the `OkHttpClientHelper`, then we can do it there.

Reply:
By now I believe that there is no good reason to install the proxy authenticator globally, but not the non-proxy authenticator. So yes, I'd move it to the `OkHttpClientHelper`.